src/cryptoadvance/specterext/devhelp/service.py

```diff
@@ -26,6 +27,8 @@ class DevhelpService(Service):
             "app": app,
             "flask": flask,
             "flask_login": flask_login,
+            "current_user": current_user,
```

Comment:
This doesn't work:
![image](https://user-images.githubusercontent.com/60378539/186642227-8302300f-ad22-4d9d-a0ad-88f8c235257b.png)

It is accessible via:
![image](https://user-images.githubusercontent.com/60378539/186642289-413746f9-cdd7-4141-a2cf-c73aed34eee3.png)

Reply:
Fxed!